test/tap/LoggingEvent-test.js

```diff
@@ -106,6 +106,7 @@ test('LoggingEvent', (batch) => {
     t.equal(event2.lineNumber, undefined);
     t.equal(event2.columnNumber, undefined);
     t.equal(event2.callStack, undefined);
+    t.equal(event.functionName, undefined);
```

Comment:
Tests didn't pass due to typo.
```suggestion
    t.equal(event2.functionName, undefined);
```

Reply:
Fixed.